english/exercise/exercise.md

```diff
@@ -0,0 +1 @@
+If you [eat too much mashmallows](https://github.com/udacity/create-your-own-adventure/blob/master/english/eating-walls/eating-marshmallows.md), you will gain weight. Therefore, be sure to go for a run and do some exercise, lose some calories and then [eat](https://github.com/udacity/create-your-own-adventure/blob/master/english/eating-walls/eating-marshmallows.md) again.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.